eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -98,6 +99,16 @@ object Channel {
    */
   case class OutgoingMessage(msg: LightningMessage, peerConnection: ActorRef)
 
+  def replayPendingFailsAndFulfills(sendTo: ActorRef, pendingRelay: PendingRelayDb, channelId: ByteVector32, sendSignCmd: Boolean)(log: LoggingAdapter): Unit = {
```

Comment:
I don't know if we're ready to put this kind of helper function in the object instead of the class.
It feels a premature, that's probably something we'll do when we move this actor to akka typed.